app.js

```diff
@@ -2,15 +2,28 @@
 const express = require("express");
 const app = express();
 const { getTopics } = require("./controllers/topics-controller.js");
+const { getArticles } = require("./controllers/articles-controller.js");
 
 //ENDPOINTS
 app.get("/api/topics", getTopics);
+app.get("/api/articles/:article_id", getArticles);
 
-//ERROR HANDLING 
+//ERROR HANDLING
 app.use((err, req, res, next) => {
-    res.status(500).send({msg: `Sorry! We can't do that right now`})
-})
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: `That input is invalid` });
+  }
+  next(err);
+});
 
+app.use((err, req, res, next) => {
+  res.status(404).send({ msg: `We don't have that article!` });
+});
```

Comment:
In this error handler you're not invoking `next(err)` so it can't ever get to the `500` error handler, which could cause you some headaches if you encounter an unexpected error while developing an endpoint. Maybe set up some `if` logic with an `else` so you can invoke _next_